helm.yml

```diff
@@ -12,15 +12,19 @@ plugins:
   - flant-statusmap-panel
   - simpod-json-datasource
   - grafana-piechart-panel
+datasources:
+  datasources.yaml:
+    apiVersion: 1
+    datasources:
+    - name: "Elasticsearch - Logging"
+      type: elasticsearch
+      orgId: 3
+      readOnly: false
+      editable: true
 admin:
   existingSecret: grafana-admin-user
   userKey: admin-user
   passwordKey: admin-password
-extraVolumeMounts:
-  - name: provisioning-datasources
-    mountPath: /etc/grafana/provisioning/datasources
-    readOnly: false
-    existingClaim: provisioning-datasources
```

Comment:
The task that creates this existing claim will need removing from the pipeline, it can be found [here](https://github.com/SkillsFundingAgency/das-grafana/blob/a0ab269599f414dbeb5c0860b3c153278501bb29/pipeline-templates/job/deploy-grafana.yml#L107) and the manifest file associated with it is [here](https://github.com/SkillsFundingAgency/das-grafana/blob/master/manifest/volume-claim.yml)

Reply:
Cheers @NickGraham101 I've deleted the volume-claim.yml and removed the deployment step. Will test the deployment in DEV.